app/scss/components/_grid.scss

```diff
@@ -166,7 +166,14 @@
   @include flex(0, 0, 90%);
   max-width: 90%;
 }
-
+.left {
```

Comment:
@bl4ckdu5t you're right. I will read them. I just looked in the gulpfile and I saw that bootstrap is not used any more. I'll have a deeper look in the repository next week when I'll have a laptop again :D